databroker/broker.py

```diff
@@ -849,7 +849,7 @@ def export(self, headers, db, new_root=None, copy_kwargs=None):
             an instance of databroker.Broker class, which has
             filestore (fs) and metadatastore (mds) attributes
             that will be the target to export info
-        new_root : str
+.        new_root : str
```

Comment:
Is it a random dot, or some undocumented trick?

Reply:
that as a random dot